src/python/ensembl/io/genomio/genbank/get_genbank.py

```diff
@@ -33,48 +34,47 @@ def __init__(self, msg):
         self.msg = msg
 
 
-class InputSchema(argschema.ArgSchema):
-    """Input arguments expected by this script."""
-
-    accession = argschema.fields.String(
-        metadata={"required": True, "description": "Sequence accession required"}
-    )
-
-
-def download_genbank(accession: str) -> str:
+def download_genbank(accession: str, output_gb: PathLike) -> None:
```

Comment:
```suggestion
def download_from_ncbi(accession: str, output_gb: PathLike) -> None:
```
I may be wrong, but if you are using Entrez doesn't that mean that you can fetch from Genbank as well as RefSeq? Thus, maybe NCBI is more accurate in this case.

Reply:
This particular piece of code only looks at genbank records via nuccore, so unless we rewrite it to support only things it should be fine as is